code/game/objects/items/storage/pouch.dm

```diff
@@ -118,23 +118,24 @@
 	name = "first-aid pouch"
 	desc = "Standard marine first-aid pouch. It can contain autoinjectors, sets of pills, and bandages."
 	icon_state = "firstaid"
-	storage_slots = 6
+	storage_slots = 7
 	can_hold = list(
 		/obj/item/reagent_containers/hypospray/autoinjector,
 		/obj/item/stack/medical,
 		/obj/item/storage/pill_bottle,
```

Comment:
packet only now then

Reply:
I don't see how it needs to be only packets considering medkit pouches are the same storage.